.github/CODEOWNERS

```
@@ -1,3 +1,48 @@
 # https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
 
-* @ux-tools-team-tools @ux-tools-team-elements
+# Module: store
+/packages/store @SAP/ux-tools-store
+
+# Module: fe-fpm-writer
+/packages/fe-fpm-writer/ @SAP/ux-tools-fpm-writer
```

To follow the naming convention, shouldn't it be `@SAP/ux-tools-fe-fpm-writer`?
```suggestion
/packages/fe-fpm-writer/ @SAP/ux-tools-fe-fpm-writer
```

---

@Klaus-Keller Thanks for the feedback. I have updated the team names as you have suggested.
Initial mismatch was as I was thinking to keeping the names a bit shorter, but I get your point. Lets be consistent. 
https://github.com/SAP/open-ux-tools/commit/de4667c65a5badfb9d1a9212d326df1d989ed782